ApollosProject/apollos-apps
# fix: export ApollosPlayerConnected; replace GET_MEDIA import with query prop

## ✏️ Description
While setting up Community Christian I noticed that NodeSingleConnectedWithMedia would break each time I navigated to a specific item with media.

With the move of some of the ui-connected pkg contents to templates it looks like we now need to:
- export ApollosPlayerConnected from apollos-ui-connected/src/index
- pass GET_MEDIA as a prop now that it resides in templates and is required in pkgs -> ui-connected -> ApollosPlayerConnected



## 🔬 To Test
1. Spin up a one of the multitenant apps and view a content item that has media
- you should see an error immediately (see screenshots)
2. Checkout branch fix-NodeSingleConnectedWithMedia and test out the same app again
- the item should load and video playback should work

## 📸 Screenshots
**Before**
<img width="1728" alt="Screen Shot 2022-11-04 at 15 44 26" src="https://user-images.githubusercontent.com/19194337/200061857-beb10a15-97b5-40fb-be44-30376b1494c4.png">

**After**
<img width="1728" alt="Screen Shot 2022-11-04 at 15 47 01" src="https://user-images.githubusercontent.com/19194337/200062252-76ed746e-2734-4a28-b270-f825ad360d73.png">

## ⚠️ To-do Before Merge



## 📝 Checklist



- [] New and existing tests and linting pass locally with my changes
